__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require('supertest');
+const app = require('../app')
+
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => seed(testData));
+
+
+describe('GET /api/topics', ()=>{
+    test('200: responds with an array of topic objects, each of which should have the following properties: slug and description', ()=>{
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((res)=>{
+                console.log(res.body, '<<<res.body')
+                expect(res.body).toBeInstanceOf(Object);
+                expect(res.body).not.toBeInstanceOf(Array);
```

Comment:
have a look and see if you need both of these toBeInstanceOf tests!

Reply:
removed to be not instance of array as all arrays are objects